Source/WebKit/UIProcess/ios/WKScrollView.mm

```diff
@@ -464,6 +470,25 @@ - (void)_updateScrollability
     [super setScrollEnabled:(_scrollEnabledByClient && _scrollEnabledInternal)];
 }
 
+- (void)setBounces:(BOOL)value
+{
+    _bouncesSetByClient = value;
```

Comment:
I know this follows existing naming, but I find that naming confusing. There's ambiguity about whether this is "the value set by the client" or "the client has set some value". "set by client" implies the second. I'd prefer we rename them all to _clientBouncesValue or something.

Reply:
It would be clearer if these were std::optional<> too.

Reply:
Agreed. This probably warrants a follow up patch.